pkg/ccl/changefeedccl/sink_webhook_v2.go

```diff
@@ -243,6 +297,25 @@ func (sc *webhookSinkClient) CheckConnection(ctx context.Context) error {
 	return nil
 }
 
+func (sc *webhookSinkClient) setRequestHeaders(req *http.Request) {
+	switch sc.format {
+	case changefeedbase.OptFormatJSON:
+		req.Header.Set("Content-Type", applicationTypeJSON)
```

Comment:
nit: maybe add a constant for `Content-Type` too?

Reply:
Added